src/modules/umount/main.py

```diff
@@ -47,6 +49,13 @@ def run():
     :return:
     """
     root_mount_point = libcalamares.globalstorage.value("rootMountPoint")
+    log_source = libcalamares.job.configuration["srcLog"]
+    log_destination = libcalamares.job.configuration["destLog"]
+    
+    # copy installation log before umount
+    if(os.path.exists('{!s}'.format(log_source))):
```

Comment:
Can you please just add another if to check whether srcLog is set (is not empty)? That way if the deployer doesn't wish to copy the log they simply don't set srcLog.